extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -14,335 +14,394 @@
  * limitations under the License.
  */
 
-@import './amp-story-share.css';
-
-
-.i-amphtml-story-system-layer {
-  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.35), transparent) !important;
-  position: absolute !important;
-  top: 0 !important;
-  left: 0 !important;
-  right: 0 !important;
-  height: 56px !important; /* 6px progress bar + 48px icons + 2px margin */
-  z-index: 100000 !important;
-  box-sizing: border-box !important;
-  transition: opacity 0.3s !important;
-  pointer-events: none !important;
-}
-
-.i-amphtml-story-bookend-active.i-amphtml-story-system-layer {
-  opacity: 0.3 !important;
-}
-
+ @import './amp-story-share.css';
```

Comment:
Hmm it looks like there's a problem with the diff in this file. It's showing as if you modified the entire file.

Can you try to make it show only the changes you did?

Reply:
Yeah I noticed that too. I'll look into it!